### PR TITLE
Issue2446 fix data centers - Updates from Comments

### PR DESCRIPTION
**Pull request by khinkelman**
Hi @AntoineGautier, here are the updates from your comments in PR #2455. Please let me know if you need anything else. Thanks!

### CODE REVIEW

File: Buildings/Applications/DataCenters/ChillerCooled/Equipment/BaseClasses/PartialPlantParallel.mo [hunk 1]
```diff
@@ -53,14 +49,14 @@ partial model PartialPlantParallel
     each final m_flow_nominal=m1_flow_nominal,
     each dpFixed_nominal=dp1_nominal,
     each final show_T=show_T,
+    each final kFixed=m1_flow_nominal/sqrt(dp1_nominal),
```
Comment (AntoineGautier) [inline Buildings/Applications/DataCenters/ChillerCooled/Equipment/BaseClasses/PartialPlantParallel.mo:52]: Please remove that binding (also for `val2`). This is redundant with the binding for `dpFixed_nominal`. There seems to be an inconsistency in the valve model parameters, see https://github.com/ibpsa/modelica-ibpsa/issues/1472. (Sorry I just realize that now.)

**Review by AntoineGautier — CHANGES_REQUESTED** [review]
Please see my inline comment for the binding of `kFixed`. 
I believe everywhere it is used, it should be removed in favor of the binding of `dpFixed_nominal` (which is already there most of the time, introducing a redundancy). See https://github.com/ibpsa/modelica-ibpsa/issues/1472.
Could you please make the changes in all the following instances?
```
(base) ~/gitrepo/modelica-buildings[issue2446_fixDataCenters]$ grep -nr 'kFixed=' Buildings/.
Buildings/./Applications/DataCenters/ChillerCooled/Equipment/BaseClasses/PartialIntegratedPrimary.mo:65:    final kFixed=0,
Buildings/./Applications/DataCenters/ChillerCooled/Equipment/BaseClasses/PartialIntegratedPrimary.mo:87:    final kFixed=0,
Buildings/./Applications/DataCenters/ChillerCooled/Equipment/BaseClasses/PartialCoolingCoilHumidifyingHeating.mo:176:      final kFixed=kFixed,
Buildings/./Applications/DataCenters/ChillerCooled/Equipment/BaseClasses/PartialPumpParallel.mo:40:  parameter Real kFixed=m_flow_nominal/sqrt(dpValve_nominal)
Buildings/./Applications/DataCenters/ChillerCooled/Equipment/BaseClasses/PartialPumpParallel.mo:128:    each final kFixed=kFixed,
Buildings/./Applications/DataCenters/ChillerCooled/Equipment/BaseClasses/PartialPlantParallel.mo:38:    each final kFixed=kFixed[2],
Buildings/./Applications/DataCenters/ChillerCooled/Equipment/BaseClasses/PartialPlantParallel.mo:63:    each final kFixed=kFixed[1],
Buildings/./Applications/DataCenters/ChillerCooled/Equipment/BaseClasses/PartialChillerWSE.mo:184:    final kFixed=kFixedValChi,
Buildings/./Applications/DataCenters/ChillerCooled/Equipment/IntegratedPrimarySecondary.mo:105:    final kFixed=0,
Buildings/./Applications/DataCenters/ChillerCooled/Equipment/IntegratedPrimarySecondary.mo:133:    final kFixed=kFixedValPum,
Buildings/./Applications/DataCenters/ChillerCooled/Equipment/IntegratedPrimaryLoadSide.mo:87:    final kFixed=kFixedValPum,
Buildings/./Applications/DataCenters/ChillerCooled/Equipment/WatersideEconomizer.mo:7:    final kFixed={m1_flow_nominal,m2_flow_nominal} ./ sqrt({dp1_nominal,dp2_nominal}),
```

File: Buildings/Applications/DataCenters/ChillerCooled/Equipment/BaseClasses/PartialHeatExchanger.mo [hunk 2]
```diff
@@ -64,7 +64,7 @@ partial model PartialHeatExchanger "Partial model for heat exchangers "
     final R=R,
     final delta0=delta0,
     final fraK=fraK_ThrWayVal,
-    final dpFixed_nominal={dp2_nominal,0},
+    final dpFixed_nominal={if activate_ThrWayVal then dp2_nominal else 0, 0},
```
[the remark this answers is not in the document]
Comment (khinkelman) [reply Buildings/Applications/DataCenters/ChillerCooled/Equipment/BaseClasses/PartialHeatExchanger.mo:67]: You are correct. I must have saved the revision on accident. Thanks for catching this!

File: Buildings/Applications/DataCenters/ChillerCooled/Equipment/BaseClasses/PartialPlantParallel.mo (hunk 1 above)
Comment (khinkelman) [reply Buildings/Applications/DataCenters/ChillerCooled/Equipment/BaseClasses/PartialPlantParallel.mo:52]: Removed for `val1` and `val2`.

**Comment by khinkelman** [comment]
Hi @AntoineGautier, thanks again for your review and comments. I believe I have address them all, and I also verified that `grep -nr 'kFixed=' Buildings/.` no longer produces any results. Please let me know if there is anything else you need. Thanks!

**Pull request merged**